Validator/Constraints/AddressValidator.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BazingaGeocoderBundle package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Bazinga\GeocoderBundle\Validator\Constraints;
```

Comment:
It is good practice to always have namespaces and classes in singular. 

`namespace Bazinga\GeocoderBundle\Validator\Constraint; `